src/Shared/LogMessagePacketBase.cs

```diff
@@ -800,6 +813,13 @@ private void WriteBuildMessageEventToStream(BuildMessageEventArgs buildMessageEv
             translator.TranslateEnum(ref importance, (int)importance);
         }
 
+        private void WriteResponseFileUsedEventToStream(ResponseFileUsedEventArgs responseFileUsedEventArgs, ITranslator translator)
+        {
+            // code code code code
+            MessageImportance importance = MessageImportance.Normal;
```

Comment:
Do we need to translate an importance for this? It doesn't look like we need it at deserialization time.

Reply:
AFAIK, `ResponseFileUsedEventArgs` only contains information on the response file path, so I am not sure what else should be translated in this method